src/Utilities/CMakeLists.txt

```diff
@@ -13,3 +13,7 @@
 #\todo move qmcutils here
 
 subdirs(for_testing)
+
+if(BUILD_UNIT_TESTS)
```

Comment:
This always makes me imagine a developer testing their new code using the application. If you are going through enough recompiles that this is a good option, write a test.

Is there another use case?

Reply:
BUILD_UNIT_TESTS was introduced to allow disabling building unit test to speed-up compilation. I never turned it off.
Since it was there, I just keep it around as I move bits in CMake.

Reply:
Ok we might want to revisit it in a all hands meeting someday.